### PR TITLE
Updated marketing page and support for registration

### PR DESCRIPTION
**Pull request by devyoustice**
This is submitted for early feedback for validation and approval.
**NOTE: Don't merge this release until further notice.**

Special questions:
* can/should we use embedded fonts (as it is currently)?

Known limitations: 
* The registration form won't create real accounts

### CODE REVIEW

**Comment by devyoustice** [comment]
Any feedback is really appreciated. The sooner, the better.

**Comment by Quetzacoalt91** [comment]
Hi @devyoustice,

Including new fonts is allowed. Just take care to store them in the folder *views/fonts* as you did and add an index.php file. Also, make sure they work in HTTP and HTTPS.

**Comment by Quetzacoalt91** [comment]
Your class Translator seems to be a duplicate from the translation system of PrestaShop. You can set your Youtube URLs as a translated text.
Here is an example: https://github.com/PrestaShop/gadwords/blob/master/fr.php#L7

Also, describing why you need all this data during the registration would be appreciated.

cc @sdesbrosses

**Comment by devyoustice** [comment]
Those data are collected essential to carry out the registration. We will add some explanation in a future version, but for now let's do without. As for the translator class, yes it is pretty much a duplicate. The reason why it's part of the SDK is that we need it in other environments, or custom integrations and would like to minimize the differences between the generic SDK and the one included in the this module. Is this OK?
URLS: OK, will be set as translatable text.

**Comment by devyoustice** [comment]
Commit https://github.com/devyoustice/yousticeresolutionsystem/commit/af6fe829b068222836d80191dcb09e72ce43a606 is mergeable and publishable.

**Comment by devyoustice** [comment]
Hi, could you please give any eta, when our submission is going to get reviewed?

**Comment by Quetzacoalt91** [comment]
Hi,

I plan to review modules tomorrow morning. You should have some news at this time.

Best regards

**Comment by devyoustice** [comment]
Thanks. Does it mean, that (if successfully validated), the module will immediately be available and published by you, or is that a separate process?

**Comment by Quetzacoalt91** [comment]
It is a separate process executed each night.

**Comment by devyoustice** [comment]
hi, any news?

File: views/templates/admin/main.tpl [hunk 1]
```diff
@@ -22,168 +22,438 @@
...
+        <p>{l s='Youstice is designed to resolve customer disputes in a quick, emotion-free way, avoiding lengthy conversations or negative sentiment across social networks. Shoppers express their issue and propose a solution, choosing from options directly through the application. Your staff then responds and negotiates a win-win resolution, collecting valuable insight on how to improve the quality of your services.'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</p>
+
+        {if $languageCode == 'fr'}
+            <div class="col col-50 no-bottom-margin">
+                <img src="{$modulePath|escape:'htmlall':'UTF-8'}views/img/girl-fr.jpg"/>
+                <h2>{l s='RESOLVE CUSTOMER COMPLAINTS WITH YOUSTICE'|escape:'htmlall':'UTF-8' mod='yousticeresolutionsystem'}</h2>
+            </div>
+            <div class="col col-50 no-bottom-margin">
+                <img src="{$modulePath|escape:'htmlall':'UTF-8'}views/img/notebook-fr.jpg"/>
+                <h2>COUVREZ LES COÛTS DE MÉDIATION AVEC YOUSTICE</h2>
```
Comment (Quetzacoalt91) [inline views/templates/admin/main.tpl]: Even if we can find ```{if $languageCode == 'fr'}```, you should add the translation tags around the displayed text.

**Comment by devyoustice** [comment]
Would it be an acceptable compromise this time? We will resolve it later, however, in due time.

**Comment by Quetzacoalt91** [comment]
We will see if you change it next time :smile:

**Comment by devyoustice** [comment]
Question: if we added the translation tags, it would show up in the translation UI, meaning, translators (person) would mistakenly assume, that these texts also would need to be translated into other languages, which, however, is not the case, and thus we can save translation costs and avoid misunderstandings.

**Comment by Quetzacoalt91** [comment]
Having this ```{if $languageCode == 'fr'}``` is in my opinion a temporary fix for displaying content in your views. That's why I consider this part will be removed later, which makes this translation tags mandatory.

Best regards

**Comment by devyoustice** [comment]
that is correct. Thanks for the clarification.

**Pull request merged**